core/src/program.rs

```diff
@@ -201,10 +212,48 @@ impl<EC: EvalCache> Program<EC> {
         self.vm.eval_full(t, &initial_env).map_err(|e| e.into())
     }
 
-    /// Same as `eval`, but proceeds to a full evaluation.
-    /// Skips record fields marked `not_exported`
-    pub fn eval_full_for_export(&mut self) -> Result<RichTerm, Error> {
-        let (t, initial_env) = self.prepare_eval()?;
+    /// Same as `eval`, but proceeds to a full evaluation. Optionally take a set of overrides that
+    /// are to be applied to the term (in practice, to be merged with).
+    ///
+    /// Skips record fields marked `not_exported`.
+    ///
+    /// # Arguments
+    ///
+    /// - `override` is a list of overrides, where the first element is a field path represented as an
+    ///   array of strings, and the second is Nickel source code as string. Each override is imported
```

Comment:
```suggestion
    /// - `override` is a list of overrides in the form of an iterator of [`FieldOverride`]s. Each override is imported
```
Something like that. It's not a list of pairs anymore since we introduced `FieldOverride`.